### PR TITLE
[WIP] Add Basemaps documentation (install and style customization)

### PR DESCRIPTION
**Pull request by yjacolin**
Ping @Nicolasribot ;)

### CODE REVIEW

File: en/basemaps/style.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+This page contains information about the way to tweak map styles using the
+mapserver/basemaps project.
+
+Prior to using this version of mapserver/basemaps, you have to import OSM map
+data into a postgis database using the imposm tool. See :ref:`basemaps_install`
+to know how to set up easily your project.
+
+Generating a mapfile
+=====================
+
```
Comment (havatv) [inline en/basemaps/style.txt]: Does not seem complete.

Comment (jmckenna) [reply en/basemaps/style.txt]: agreed, the file 'style.txt' seems to be not finished.

Comment (yjacolin) [reply en/basemaps/style.txt]: Yes, this is a Work in Progress. I should fill this file from Nicolas notes.

**Comment by jmckenna** [comment]
@yjacolin can you re-do this pull request into the new branch-7-2  ?  (this is the new default / working branch as of today).  Sorry!

**Comment by yjacolin** [comment]
no problem!

**Comment by yjacolin** [comment]
superseeded by #247

**Pull request closed without merging**